src/QirRuntime/README.md

```diff
@@ -11,24 +12,17 @@ This folder contains QIR runtime project, which includes implementation of the
 
 ## Build
 
+### Prerequisites
 The QirRuntime project is using CMake (3.17) + Ninja(1.10.0) + Clang++(11.0.0). Other versions of the tools might work
- but haven't been tested. Only x64 architecture is supported.
+ but haven't been tested. Only x64 architecture is supported.  
+For runing the PowerShell scripts below use the powershell core (`pwsh`),
```

Comment:
Is this necessary? I believe with a minor tweak to the condition in prerequisites.ps1 the scripts run just fine on the out of the box installation (they did about three weeks ago in any case).
```suggestion
For running the PowerShell scripts below use the powershell core (`pwsh`),
```

Reply:
After Stefan significantly refactored our build scripts 1 - 2 weeks ago, for me the .ps1 were failing without `pwsh`.